### ISSUE TITLE
Screen reader is not announcing the Additional Feedback label name for the edit field.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5020]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Visually challenged users who rely on screen readers are getting impacted if Screen reader is not announcing the Additional Feedback label name for the edit field.

### What browsers and screen readers do this issue affect?

Others

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://airtuserexpdev.azurewebsites.net/Dashboard 

### How to reproduce the issue?

Step 1: Open the above URL in Edge Browser->SAGE chat window will display->Invoke "Chat in teams" button (open in web).
Step 2: Type “What is the accessibility process”->Invoke “Dislike” button.
Step 3: Verify whether Screen reader is announcing the Additional Feedback label name for the edit field or not.

### What do you expect?

Screen reader should announce the Additional Feedback label name for the edit field.

### What actually happened?

Screen reader is not announcing the Additional Feedback label name for the edit field.
Observation:
Screen reader is announcing placeholder value in the edit field.

### Do you have any screenshots or recordings to repro the issue?

<img width="708" alt="MAS_3 3 2_Screen reader is not announcing the Additional Feedback label name for the edit field" src="https://github.com/microsoft/BotFramework-WebChat/assets/117266902/f58eb974-e189-4f64-a579-6fdbbeebce31">

### MAS reference

https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B3752B546-30D8-44D7-8221-6FC32EEB3AC8%7D&file=MAS%203.3.2%20%25u2013%20Labels%20or%20Instructions.docx&action=default&mobileredirect=true

### WCAG reference

https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html

**Comment by Kirank915** [comment]
Bug copied from Accenture team: [BUG 11645949](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11645949)

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev3;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by OEvgeny** [comment]
@Kirank915  this looks like Teams issue. Teams doesn't use WebChat for chat
